index.php

```diff
@@ -4,22 +4,23 @@
 
 <main id="main-content">
   <section id="posts" class="container">
-    <div class="grid-row">
+    <div id="shuffle-preloader"></div>
+    <div id="shuffle-container" class="grid-row">
 
 <?php
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
 ?>
-        <article <?php post_class('grid-item item-s-12 item-m-4'); ?> id="post-<?php the_ID(); ?>">
+        <article <?php post_class('shuffle-item item-s-12 item-m-4'); ?> id="post-<?php the_ID(); ?>">
```

Comment:
line length is a bit weird at item-m-4 tablet container. maybe`item-m-6 item-l-4`  ?